### ISSUE TITLE
CIFAR-10 Benchmarks

### ISSUE THREAD

**Issue by fangkuoyu** [XiangboGaoBarry/ConvKAN-Zoo#2]
I would like to list some benchmarks on CIFAR-10 here.

- Accuracy 80% - 90% ~ simple convolutional networks with good training strategy [ref](https://machinelearningmastery.com/how-to-develop-a-cnn-from-scratch-for-cifar-10-photo-classification/)
- Accuracy 90% - 95% ~ some advanced convolutional networks, e.g., VGG/ResNet [ref](https://github.com/kuangliu/pytorch-cifar)
- Accuracy 95% + ~ some transformer networks, e.g., ViT [ref](https://github.com/kentaroy47/vision-transformers-cifar10)

So, I think that we should try to see if KAN for CIFAR-10 can reach 80% - 90% first. Trying ResNet and ImageNet is too aggressive at the current stage.

Regards

**Comment by XiangboGaoBarry** [comment]
Feel free to try enlarging the model size or using other techniques to increase accuracy on the CIFAR10 dataset. We do not plan to release CIFAR10 benchmarks for larger models, as our main purpose for this repo is to compare the differences between nn.Conv and KA-Conv. We will release KA-Conv based ResNet and other models evaluated on ImageNet or COCO. Stay tuned.
